packages/system/external-secrets-operator/Chart.yaml

```
@@ -0,0 +1,3 @@
+apiVersion: v2
+name: cozy-external-secrets-operator
+version: 0.0.0 # Placeholder, the actual version will be automatically set during the build process
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)